### DISCUSSION TITLE
[RFC]軽微な障害が起きた際のお知らせ方法について

### DISCUSSION THREAD

**Discussion by kskwtnk** [increments/qiita-discussions#29] [要望・アイデア・不具合]
クローズドベータテストに参加してくださっているみなさまだからこそコメントをいただけると嬉しいです！

Qiitaにおいて軽微な障害（繋がりづらい・数分落ちていたなど）が起きた際、どういった情報があると効果的でしょう？
大きめの障害の際は公式Twitterにて報告をしているのですが、かなり軽微なものについてのアナウンスはしていません

正直な話、大抵のサービスにおいて「少し繋がりづらい」レベルで障害報告を出すことはしていないと思います
しかしQiitaの場合かなりの数のユーザーさまにご利用いただいているのもあり、平日の日中に繋がらなくなると不安のツイートが多く投稿されるのも認識しています

いただいた意見を全て実現できるとは限らないのですが、どういう場所でどういうコミュニケーションが取れると安心できそうか、意見をいただきたいです！

---

ちなみに……

実は過去にQiitaのstatus pageが存在していたのですが、全然アクセスされていなかったためクローズしたという経緯があります

**Comment by murakami** [comment]
外から見た状況にあったステータスを表示させるのは、なかなか難しいと思います。
status pageがあることによって運用が楽になるのなら良いと思いますが、大変でしたら、現状の報告方法でも問題ないと思います。

**Comment by blono** [comment]
Qiitaが障害で落ちている時にはQiitaにはアクセスできないから大半のユーザはそこで察して時間をあけるだけかなと思いました。
（笑い話ですがGHEが落ちたときの復旧方法がGHEに書いてあって結局復旧方法がわからないとかありました、本当に切なかった😱）
QiitaのTwitterはフォローしない人かなり多いんじゃないでしょうか？
このGitHubのDiscussionsだけではなく、issueも活用して、落ちたときはissueを上げるとか、Qiitaのバックエンドがダウンしているけどフロントエンドだけは生き残っている場合には、画面にQiitaのtwitterのタイムラインを表示してみるとかどうだろうって思いました。

**Comment by uhooi** [comment]
私は繋がらなくなると障害だと察するので、特に困っていませんでした。Twitter の投稿で他の人も同じだとわかるので。
現状の運用で問題ないと感じます。

あえていうなら、公式 Twitter やステータスページは見ないので、障害発生時にエラーページへ飛ばしてエラーコードや詳細が表示されると嬉しいです。
ユーザーの特性（IT リテラシーが高いはず）上、最悪エラーコードだけでもいいかもしれません。

**Comment by kskwtnk** [comment]
みなさんありがとうございます！

実際どう感じるのだろう……と悩むことも多いため、非常に参考になります！
